### ISSUE TITLE
Javascript error on saving preferences

### ISSUE THREAD

**Issue by RicoLaaa** [ramboxapp/community-edition#2214]
### Steps to reproduce

1. Open preferences
2. Click on the "save" button
3. A JavaScript error occurred in the main process

### Expected behavior

.. no error

### Actual behavior

... error displayed

```
Uncaught Exception:
ReferenceError: appLauncher is not defined
    at EventEmitter.<anonymous> (/Applications/Rambox.app/Contents/Resources/app.asar/electron/main.js:258:23)
    at EventEmitter.emit (events.js:182:13)
    at WebContents.<anonymous> (/Applications/Rambox.app/Contents/Resources/electron.asar/browser/api/web-contents.js:346:13)
    at WebContents.emit (events.js:182:13)
```

### ENVIRONMENT

> Rambox: v0.6.6

> OS: macOS 10.14.4

> Arch: x64

**Comment by KeirDavis1** [comment]
I get the same error

**Comment by Corchoneitor** [comment]
Thanks for your report guys! Luckily, preferences are saved correctly.

**Comment by nopticon** [comment]
![rambox-error](https://user-images.githubusercontent.com/827747/56272400-27ed7d00-60ea-11e9-9d49-e1ddce84d8ff.png)

**Comment by danjames92** [comment]
Is the fix included in 0.6.6 cause I'm still seeing it here?
